### DISCUSSION TITLE
how to include fontawesome icons for mermaid diagrams

### DISCUSSION THREAD

**Discussion by gureckis** [slidevjs/slidev#1763] [Q&A] [answered]
Mermaid.js supports icons from FontAwesome (https://mermaid.js.org/syntax/flowchart.html#basic-support-for-fontawesome) but I'm a little lost about how to set that up with Slidev.  Does anyone have a hint?  I think the idea is to import some JS and CSS code but where it would go in slidev is not clear to me.

**Comment by kermanx** [comment, marked as answer]
We are tracking this at https://github.com/slidevjs/slidev/issues/1062. IIRC, I once tried to make this possible, but somehow failed.
